### PR TITLE
feat: stable scheduler

### PR DESCRIPTION
**Pull request by ianchen4work**
Add concrete implementation of stable scheduler.
Github issue #13

### CODE REVIEW

File: services/orchestra/recommendation/stable_scheduler.py [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+import operator
+import copy
+
+from framework.log.logger import Logger
+from services.orchestra.recommendation.data_processor import DataProcessor
+
+import numpy as np
+
+
+class StableScheduler(object):
```
Comment (chinming0822) [inline services/orchestra/recommendation/stable_scheduler.py]: Inherit Scheduler class?

File: services/orchestra/recommendation/stable_scheduler.py [hunk 2]
```diff
@@ -0,0 +1,169 @@
+""" concrete implementation of stable-policy scheduler  """
```
Comment (chinming0822) [inline services/orchestra/recommendation/stable_scheduler.py]: Write your algorithm descriptions here.

File: services/orchestra/recommendation/stable_scheduler.py (hunk 1 above)
Comment (ianchen4work) [reply services/orchestra/recommendation/stable_scheduler.py]: We are rethinking whether to abstract common parts of different scheduling policies into abstract class (Scheduler class). Currently, we choose to implement concrete class for each scheduling policy directly.

File: services/orchestra/recommendation/stable_scheduler.py (hunk 2 above)
Comment (ianchen4work) [reply services/orchestra/recommendation/stable_scheduler.py]: Will be provided in next commit!

File: services/orchestra/recommendation/stable_scheduler.py [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+
+        score = 0.0
+
+        for metric_name in self.config["metric_types"].values():
+
+            metric_workload = np.array([[modified_workload[node_name][metric_name][timestamp]
+                                         for timestamp in modified_workload[node_name][metric_name]]
+                                        for node_name in modified_workload.keys()])
+
+            score += self.config["weighting"][metric_name] *\
```
Comment (susansdb) [inline services/orchestra/recommendation/stable_scheduler.py]: Where's your modified configure file? It is not in your PR.

File: services/orchestra/recommendation/stable_scheduler.py [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+            pod_predicted_data: prediction workload for pods that are to be scheduled
+            node_predicted_data: prediction workload for all nodes in the cluster
+            pods_current_node: list of node that the pods currently locate in
+
+        """
+
+        pod_predicted_data = []
+        pods_current_node = []
+
+        for pod in range(len(pod_info)):
```
Comment (susansdb) [inline services/orchestra/recommendation/stable_scheduler.py]: Isn't pod in `range(len(pod_info))` a list index here?
I think `for pod in pod_info` is the correct one.

**Pull request merged**